x-pack/plugins/alerting/server/rules_client_factory.ts

```diff
@@ -58,6 +58,7 @@ export interface RulesClientFactoryOpts {
   backfillClient: BackfillClient;
   connectorAdapterRegistry: ConnectorAdapterRegistry;
   uiSettings: CoreStart['uiSettings'];
+  securityService: CoreStart['security'];
```

Comment:
I noticed a few references left for `securityPluginSetup` and `securityPluginStart` - for getting access to audit log stuff and APi key stuff.  Are we still ok going through the security plugin for that, for now?

Reply:
@pmuellr

The API Key service hasn't been made available from Core yet. See the task lists at https://github.com/elastic/kibana/issues/174578

The audit log service is available in CoreStart: https://github.com/elastic/kibana/pull/181644.

In 9.0, all the Security APIs should be accessed through Core. These things should be done before 9.0, so that backporting the future fixes doesn't become a pain. 

Hope that helps!